scrapy/utils/misc.py

```diff
@@ -73,6 +101,23 @@ def walk_modules(path):
     return mods
 
 
+def get_module_from_filepath(path):
```

Comment:
I have no strong opinion on this. Don't think it's gonna see much use, so maybe a YAGNI case. It is part of Pablo's original SEP though ;)